modules/core/src/main/java/org/apache/synapse/analytics/AnalyticsConstants.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.synapse.analytics;
+
+public class AnalyticsConstants {
+
+    /**
+     * Synapse Configurations
+     */
+    public static class SynapseConfiguration {
```

Comment:
shall we rename this to something else, we already have a class named SynapseConfiguration in synapse, this is conflict ideas.

Reply:
Done. Refactored to `SynapseConfigKeys`. This is a static inner class hence will be used as `AnalyticsConstants.SynapseConfigKeys...`.